src/Take.Blip.Builder.UnitTests/Actions/ProcessHttpActionTests.cs

```diff
@@ -27,10 +27,12 @@ public ProcessHttpActionTests()
         }
 
         public IHttpClient HttpClient { get; set; }
+        public IConfiguration Configuration { get; set; }
 
         private ProcessHttpAction GetTarget()
         {
-            return new ProcessHttpAction(HttpClient, Substitute.For<ILogger>());
+            Configuration = new ProcessHttpActionTestsConfiguration();
```

Comment:
the configuration parameter should be initilized as a "Substitute"...
later you can mock IpsDeniedOnHttpAction result....

Reply:
Done!